tests/st/utils/data.py

```diff
@@ -496,9 +496,43 @@
         'name': 'felixconfiguration1',
     },
     'spec': {
+        'chainInsertMode': 'append',
+        'defaultEndpointToHostAction': 'ACCEPT',
+        'failsafeInboundHostPorts': [
+            {'Protocol': 'tcp', 'Port': 666},
```

Comment:
We may want to add json tags to the `ProtoPort` objects in libcalico-go so that we don't have to use the go representation of the variables since the capitalization in the JSON looks a little funny.

Reply:
+1

keys should start with a lowercase